src/PowerShell/scripts/samples/WinGetPackageResourceSample.ps1

```diff
@@ -52,7 +52,7 @@ else
 {
     Write-Host "PowerToys 0.65.0 is not installed."
 
-    Invoke-DscResource @resource -Method Set
+    #Invoke-DscResource @resource -Method Set

```

Comment:
You don't want the sample to install PowerToys?

Reply:
I forgot to revert that after testing the script.